src/FlipMove.js

```diff
@@ -450,32 +475,27 @@ class FlipMove extends Component {
 
       // If the child element returns null, we need to avoid trying to
       // account for it
-      if (!childData.domNode) {
+      if (!childData.domNode || !child) {
         return;
       }
 
       childData.boundingBox = getRelativeBoundingBox({
-        childData,
-        parentData: this.parentData,
+        childDomNode: childData.domNode,
+        parentDomNode,
         getPosition: this.props.getPosition,
       });
     });
   }
 
-  computeInitialStyles(child) {
-    const noAnimationRequestedForThisEvent = (
-      (child.appearing && !this.props.appearAnimation) ||
-      (child.entering && !this.props.enterAnimation) ||
-      (child.leaving && !this.props.leaveAnimation)
-    );
-
-    if (noAnimationRequestedForThisEvent) {
-      return {};
-    }
```

Comment:
Why'd you kill this? IMO it's better to do calculations like this upfront, to simplify the conditionals further down.